src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/EventsController.cs

```diff
@@ -225,6 +194,11 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEvent cloudEvent)
                 }
             }
 
+            return await HandleEvents(after, from, to, party, source, type, size);
+        }
+
+        private async Task<ActionResult<List<CloudEvent>>> HandleEvents(string after, DateTime? from, DateTime? to, int party, List<string> source, List<string> type, int size)
```

Comment:
Could we get a more descriptive function name? retrieveAndAuthorizeEvents for instance ?

Reply:
Done :)